### ISSUE TITLE
How to make a window not realizable?

### ISSUE THREAD

**Issue by ajeecai** [andlabs/ui#371]
Hi,

I love this project since it provides native UI outlook. My question is, when I  make up a window based on it, I could resize it by dragging it edges and then the layout inside the window is shifted which looks ugly. How could I prevent this? I check the ui and libui header files and repository issues, there is no answer.

Thanks.

PS. Is there any confirmation message box widget, which has two buttons - "YES" or "NO" ?

**Comment by andlabs** [comment]
I haven't implemented either yet, sorry. But what do you mean by resizing it makes it look upgly?

**Comment by ajeecai** [comment]
I mean, if I put widgets in grid, they are in position like this:
<pre>
| button 1   |    text1     |
</pre>

Then I resize the edge, button 1 and text 1 are far away.
<pre>
| button 1  |                                                    text1              |
</pre>
Or 
<pre>
| button 1  |   text1                                                               |
</pre>
Or, I have dynamically set long text, the outlook would look strange
<pre>
| button 1  |  text1text1text1text1text1text1        |  
| button 2  |  text2                                 |
</pre>

here I perfer being  "text1text1..." instead

**Comment by andlabs** [comment]
And what type of control is text1?

**Comment by ajeecai** [comment]
ui.NewLabel. Whatever control type, resizing window will make it float I guess ...

**Comment by andlabs** [comment]
Right, uiLabel doesn't have wrapping or ellipses yet either, sorry about that.

**Comment by ajeecai** [comment]
Thank you andlabs
